src/main/java/org/jolokia/docker/maven/util/PomLabel.java

```diff
@@ -0,0 +1,113 @@
+package org.jolokia.docker.maven.util;
+/*
+ * 
+ * Copyright 2014 Roland Huss
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import java.util.UUID;
+
+/**
+ * Label used to mark a container belonging to a certain build.
+ *
+ * @author roland
+ * @since 31/03/15
+ */
+public class PomLabel {
+
+    // Environment variable used to label containers
+    public static final String CONTAINER_LABEL_NAME = "docker.maven.plugin.container";
+    
+    private String mavenCoordinates;
+    private String runId;
+
+    /**
+     * Construct from a given label
+     *
+     * @param label label as stored with the container
+     */
+    public PomLabel(String label) {
+        String[] parts = label.split(":");
+        if (parts.length != 4 && parts.length != 3) {
+            throw new IllegalArgumentException("Label '" + label +
+                                               "' has not the format <group>:<artifact>:<version>[:<runId>]");
+        }
+        mavenCoordinates = parts[0] + ":" + parts[1] + ":" + parts[2];
+        runId = parts.length == 4 ? parts[3] : null;
```

Comment:
Is there a reason why runId is here null by default ? That's inconsistent to the constructor with no runId which initializes the run id to an UUID.